src/main/java/org/cascadebot/cascadebot/commands/developer/GuildCommand.java

```diff
@@ -46,7 +46,7 @@ public String command() {
 
     @Override
     public String description() {
-        return "interact with the guild";
+        return "Interact with guilds";
```

Comment:
Same

Reply:
grammar

Reply:
What are you on about? Grammar has nothing to do with what I'm saying...

Reply:
but i made the little i be a big I

Reply:
```suggestion
        return "Allows bot administrative actions to be run on guilds";
```